### PR TITLE
Update flag-changes.mdx

### PR DESCRIPTION
**Pull request by turacma**
correct JS event is `update` not `change`

Tested using `change` and wasn't seeing any updates. Found this old issue (https://github.com/launchdarkly/node-server-sdk/issues/133) and noted that it mentioned using `update`. Confirmed that this in fact does work.

### CODE REVIEW

**Comment by mmrj** [comment]
Thank you for submitting this @turacma ! We're double-checking with our SDK team and expect merge after that.

File: src/content/topics/sdk/features/flag-changes.mdx [hunk 1]
```diff
@@ -456,7 +456,7 @@ To subscribe to all feature flag changes, register listeners for change events f
 <CSTab label="JavaScript">
 
 ```js
-client.on('change', (settings) => {
+client.on('update', (settings) => {
```
Comment (yusinto) [inline src/content/topics/sdk/features/flag-changes.mdx:459]: The js sdk does use the 'change' event and not 'update', different from electron so please rollback these changes. Electron uses 'update'.

File: src/content/topics/sdk/features/flag-changes.mdx [hunk 2]
```diff
@@ -493,7 +493,7 @@ To subscribe to all feature flag changes, register listeners for change events f
 <CSTab label="JavaScript">
 
 ```js
-client.on('change', allChanges => {
+client.on('update', allChanges => {
```
Comment (yusinto) [inline src/content/topics/sdk/features/flag-changes.mdx:496]: The node js client-side sdk uses the js common sdk so uses 'change' as well. Please also rollback these thank you.

**Review by yusinto — CHANGES_REQUESTED** [review]

File: src/content/topics/sdk/features/flag-changes.mdx (hunk 1 above)
Comment (turacma) [reply src/content/topics/sdk/features/flag-changes.mdx]: Updated

File: src/content/topics/sdk/features/flag-changes.mdx (hunk 2 above)
Comment (turacma) [reply src/content/topics/sdk/features/flag-changes.mdx]: Updated

**Review by yusinto — APPROVED** [review]

**Review by mmrj — APPROVED** [review]

**Pull request merged**